manim/_config/utils.py

```diff
@@ -743,10 +1037,119 @@ def use_js_renderer(self, val):
     media_dir = property(
         lambda self: self._d["media_dir"],
         lambda self, val: self._set_dir("media_dir", val),
-        doc="Main output directory, relative to execution directory.",
+        doc="Main output directory.  See :meth:`ManimConfig.get_dir`.",
     )
 
     def get_dir(self, key, **kwargs):
+        """Resolve a config option that stores a directory.
+
+        Config options that store directories may depend on one another.  This
+        method is used to provide the actual directory to the end user.
+
+        Parameters
+        ----------
+        key : :class:`str`
+            The config option to be resolved.  Must be an option ending in
+            '_dir', for example 'media_dir' or 'video_dir'.
+
+        kwargs : :class:`str`
+            Any strings to be used when resolving the directory.
+
+        Returns
+        -------
+        :class:`pathlib.Path`
+            Path to the requested directory.  If the path resolves to the empty
+            string, return None instead.
+
+        Raises
+        ------
+        :class:`KeyError`
+            When ``key`` is not a config option that stores a directory and
+            thus :meth:`~ManimConfig.get_dir` is not appropriate; or when
+            ``key`` is appropriate but there is not enough information to
+            resolve the directory.
+
+        Notes
+        -----
+        Standard :meth:`str.format` syntax is used to resolve the paths so the
+        paths may contain arbitrary placeholders using f-string notation.
+        However, these will require ``kwargs`` to contain the required values.
+
+        Examples
+        --------
+
+        The value of ``config.tex_dir`` is ``{media_dir}/Tex`` by default,
+        i.e. it is a subfolder of wherever ``config.media_dir`` is located.  In
+        order to get the *actual* directory, use :meth:`~ManimConfig.get_dir`.
+
+        .. code-block:: python
+
+            >>> from manim import config
+            >>> config.tex_dir
+            '{media_dir}/Tex'
+            >>> config.media_dir
+            './media'
+            >>> config.get_dir("tex_dir")
+            PosixPath('media/Tex')
```

Comment:
Here is one possible way to make this test OS-agnostic:

```suggestion
            >>> config.get_dir("tex_dir").as_posix()
            'media/Tex'
```

Reply:
Ah good one!